pkg/agent/datamodel/types.go

```diff
@@ -616,55 +616,30 @@ type SysctlConfig struct {
-	PlatformFaultDomainCount            *int                 `json:"platformFaultDomainCount"`
-	PlatformUpdateDomainCount           *int                 `json:"platformUpdateDomainCount"`
-	SinglePlacementGroup                *bool                `json:"singlePlacementGroup,omitempty"`
-	VnetCidrs                           []string             `json:"vnetCidrs,omitempty"`
-	PreserveNodesProperties             *bool                `json:"preserveNodesProperties,omitempty"`
-	WindowsNameVersion                  string               `json:"windowsNameVersion,omitempty"`
-	EnableVMSSNodePublicIP              *bool                `json:"enableVMSSNodePublicIP,omitempty"`
-	LoadBalancerBackendAddressPoolIDs   []string             `json:"loadBalancerBackendAddressPoolIDs,omitempty"`
-	CustomVMTags                        map[string]string    `json:"customVMTags,omitempty"`
-	DiskEncryptionSetID                 string               `json:"diskEncryptionSetID,omitempty"`
-	UltraSSDEnabled                     *bool                `json:"ultraSSDEnabled,omitempty"`
-	EncryptionAtHost                    *bool                `json:"encryptionAtHost,omitempty"`
-	ProximityPlacementGroupID           string               `json:"proximityPlacementGroupID,omitempty"`
-	CustomKubeletConfig                 *CustomKubeletConfig `json:"customKubeletConfig,omitempty"`
-	CustomLinuxOSConfig                 *CustomLinuxOSConfig `json:"customLinuxOSConfig,omitempty"`
+	Name                   string               `json:"name"`
+	Count                  int                  `json:"count"`
+	VMSize                 string               `json:"vmSize"`
+	OSDiskSizeGB           int                  `json:"osDiskSizeGB,omitempty"`
+	KubeletDiskType        KubeletDiskType      `json:"kubeletDiskType,omitempty"`
+	DNSPrefix              string               `json:"dnsPrefix,omitempty"`
+	OSType                 OSType               `json:"osType,omitempty"`
+	Ports                  []int                `json:"ports,omitempty"`
+	AvailabilityProfile    string               `json:"availabilityProfile"`
+	ScaleSetPriority       string               `json:"scaleSetPriority,omitempty"`
+	ScaleSetEvictionPolicy string               `json:"scaleSetEvictionPolicy,omitempty"`
+	StorageProfile         string               `json:"storageProfile,omitempty"`
+	DiskSizesGB            []int                `json:"diskSizesGB,omitempty"`
+	VnetSubnetID           string               `json:"vnetSubnetID,omitempty"`
+	Subnet                 string               `json:"subnet"`
+	Distro                 Distro               `json:"distro,omitempty"`
+	CustomNodeLabels       map[string]string    `json:"customNodeLabels,omitempty"`
+	PreprovisionExtension  *Extension           `json:"preProvisionExtension"`
+	KubernetesConfig       *KubernetesConfig    `json:"kubernetesConfig,omitempty"`
+	AvailabilityZones      []string             `json:"availabilityZones,omitempty"`
```

Comment:
I don't think we're using this anywhere are we?

Reply:
Thanks for the info. I will go through them and remove unused ones in future PRs.